Application/EdFi.Ods.Features/Container/Modules/ProfilesModule.cs

```diff
@@ -79,10 +79,6 @@ public override void ApplyConfigurationSpecificRegistrations(ContainerBuilder bu
                 .As<IAdminProfileNamesPublisher>()
                 .SingleInstance();
 
-            builder.RegisterType<ProfileMetadataCacheExpiredNotificationHandler>()
```

Comment:
Handlers are registered [here](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS/blob/7c8337a40b34c13825388f90545773a3b8c60acb/Application/EdFi.Ods.Api/Startup/OdsStartupBase.cs#L242). 
This change prevents registering a duplicated handler implementation.